packages/api/src/platforms/vtex/resolvers/validateSession.ts

```diff
@@ -154,6 +154,9 @@ export const validateSession = async (
       : null,
     geoCoordinates,
```

Comment:
I don't think we should update the geoCoordinates using the b2b saved address values, since it's optional and might contain invalid data (there's no validation, as far as I noticed). I believe it's safer to rely on the received postalCode instead.